TestTools/StreamChatTestTools/SpyPattern/Spy/Logger_Spy.swift

```diff
@@ -7,8 +7,8 @@ import Foundation
 
 final class Logger_Spy: Logger, Spy {
     var originalLogger: Logger?
-    var recordedFunctions: [String] = []
-    var failedAsserts: Int = 0
+    @Atomic var recordedFunctions: [String] = []
```

Comment:
`@Cached` calls assert which ends up here and the spy object is not thread-safe.